flow-webpush/src/main/java/com/vaadin/flow/server/webpush/WebPushMessage.java

```diff
@@ -48,9 +73,21 @@ public String toString() {
      * @return JSON representation of this message
      */
     public String toJson() {
-        JsonObject json = Json.createObject();
+        ObjectNode json = objectMapper.createObjectNode();
         json.put("title", title);
-        json.put("body", body);
-        return json.toJson();
+        if (options != null) {
+            json.set("options", options);
+        }
+        return json.toString();
+    }
+
+    private static ObjectNode getBodyOption(String body) {
+        if (body != null) {
+            ObjectNode objectNode = objectMapper.createObjectNode();
+            objectNode.put("body", body);
+            return objectNode;
+        } else {
+            return objectMapper.createObjectNode();
+        }
```

Comment:
```suggestion
        ObjectNode objectNode = objectMapper.createObjectNode();
        if (body != null) {
            objectNode.put("body", body);
        }
        return objectNode;
```
Could simplify to not have if else and only one create as the difference is if we put content or not.

Reply:
Updated, thanks